### PR TITLE
A bunch of saycode fixes

### LINKED ISSUE #2097: Xenomorphs speaking anything but hivemind languages have the first three letters cut off
Almost certainly my fault with not account for Bay having human sub-species xenomorphs, but, uh, yeah.

### PR DESCRIPTION
**Pull request by tigercat2000**
Fixes #2097 
Fixes:
 - Xenomorphs having cutoff speech
 - Brains fucking up all of say
 - Redundant say code under slimes, simple animals, xenomorphs
 - Redundant say_quotes for humans
 - All mobs will now correctly process language speech verbs
 - Broken nymph saycode

Features:
 - Xenomorphs now have custom speech verbs for common
 - Default languges for Xenomorphs makes more sense, and work
	   properly

### CODE REVIEW

**Comment by Ultimate-Chimera** [comment]
Meh... Sure... :(

**Comment by FalseIncarnate** [comment]
It seems as though every time someone touches either Diona nymphs or saycode, the chirp emote gets broken.

Can you please check if nymphs can properly emote?

**Comment by tigercat2000** [comment]
@FalseIncarnate Pretty sure I tested that, but I'll test it again.

**Comment by tigercat2000** [comment]
@FalseIncarnate Nope, doesn't work! I cannot find a reference to the chirp emote anywhere, either.

**Comment by FalseIncarnate** [comment]
Probably got wiped when Diona nymphs were made simple_animal type.

If they can at least emote, I'll re-add *chirp once this is in.

**Comment by tigercat2000** [comment]
@FalseIncarnate I actually found a couple of (insane) bugs with emotes, I'll include it in this PR with my fixes.

File: code/modules/mob/living/simple_animal/friendly/diona.dm [hunk 1]
```diff
@@ -248,43 +248,12 @@
...
+	if(stat)	return
 
-/mob/living/simple_animal/diona/say(var/message)
-	if(client)
-		if(client.prefs.muted & MUTE_IC)
-			src << "\red You cannot speak in IC (Muted)."
-			return
+	switch(act) //IMPORTANT: Emotes MUST NOT CONFLICT anywhere along the chain.
+		if("chirp")
+			message = "<B>\The [src]</B> chirps!"
```
Comment (TheDZD) [inline code/modules/mob/living/simple_animal/friendly/diona.dm]: If memory serves, this emote is supposed to play `sound/misc/nymphchirp.ogg`.

**Comment by tigercat2000** [comment]
@TheDZD / @Fox-McCloud / @Markolie Going to ask for a full review, this is starting to touch a lot more than it originally did

**Comment by Fox-McCloud** [comment]
This looks good to me.

**Pull request merged**